src/mbgl/text/collision_index.cpp

```diff
@@ -122,7 +122,7 @@ IntersectStatus CollisionIndex::intersectsTileEdges(const CollisionBox& box,
     const float tileY2 = tileEdges[3];
 
     // Check left border
-    float minSectionLength = std::min(tileX1 - x1, x2 - tileX1);
+    int minSectionLength = std::min(tileX1 - x1, x2 - tileX1);
```

Comment:
This one does not seem to be needed
```suggestion
    float minSectionLength = std::min(tileX1 - x1, x2 - tileX1);
```

Reply:
False alarm. Still investigating this one.